tailscale/client.go

```diff
@@ -386,6 +386,19 @@ func (c *Client) AuthorizeDevice(ctx context.Context, deviceID string) error {
 	return c.performRequest(req, nil)
 }
 
+// DeleteDevice deletes the device given its deviceID.
+func (c *Client) DeleteDevice(ctx context.Context, deviceID string) error {
+	const uriFmt = "/api/v2/device/%s"
+	req, err := c.buildRequest(ctx, http.MethodDelete, fmt.Sprintf(uriFmt, deviceID), map[string]bool{
+		"authorized": true,
+	})
```

Comment:
Based on the spec for the [DELETE](https://github.com/tailscale/tailscale/blob/main/api.md#delete-apiv2devicedeviceid---deletes-the-device-from-its-tailnet) endpoint, it doesn't require anything in the request body. You can just provide `nil` here instead of the map.

Reply:
👍🏼 Just did this change